src/app/(admin)/admin/_utils/transformOrganizationEditData.js

```diff
@@ -0,0 +1,41 @@
+import { FormatOfWork } from '@prisma/client';
+
+export const transformOrganizationEditData = data => {
```

Comment:
i think you may destructure data object

Reply:
Thanks for comment! Refactored the whole function.